be/src/fs/fs.cpp

```diff
@@ -64,6 +81,11 @@ StatusOr<std::unique_ptr<FileSystem>> FileSystem::CreateUniqueFromString(std::st
     if (is_s3_uri(uri)) {
         return new_fs_s3();
     }
+#ifdef USE_STAROS
+    if (is_staros_uri(uri)) {
```

Comment:
if use staros, starlet should delegate all types of storage access, including local file and remote storage, not only the ones that starts with staros://, am I understand correct?